### DISCUSSION TITLE
OBD recommendation

### DISCUSSION THREAD

**Discussion by assid2** [agronick/aa-torque#105] [Q&A]
Hey guys,
Any suggestions on  some cheap obd devices which are compatible / recommended ? Maybe even something on aliexpress ?

**Comment by Daytrip-Society** [comment]
Not at my desk right now to dive for cheaper options, but the OBD dongle by ATOTO (I bought alongside my A6 PF edition) hasn't ever given me issues. Gotta give them props, between quality and support experience I've never heard bad of them, that's basically unicorn status in the world of Chinese head units

**Comment by EngineerTunerNorway** [comment]
OBDLink mx+

**Comment by davidcharlessmith516** [comment]
Here's a couple that I tried in another thread if you're looking for some relatively inexpensive ones that might not work as fast as the more expensive ones. 
#107
